fix #74 : Push default avatar image for each theme in frontend while backend initialisation

akshatbajetha opened the pull request:
#74 

Added all the relevant profile images for each theme in the frontend (Classic, Time, Vintage, Amber, Forest, Cream) and named all the images accordingly. Added a for loop on the array of all the files to be uploaded to enhance the readability of the code.

Added appwrite's unique() method to generate unique ID's for the files, instead of hard coding them.

Fixed a typo in windows script file where it was asking for "--team-id" but it was written "--teamId" in the script file.

Changed "deploy" to "push" in windows script file as deploy has been removed.

![image](https://github.com/user-attachments/assets/09623bac-339c-4409-b238-257ab4a1d908)


Please let me know if you need any more adjustments!

Aarush-Acharya commented on init.ps1:
```diff
@@ -74,11 +74,19 @@ appwrite project create-variable --key APPWRITE_API_KEY --value $secret
+
+foreach ($file in $files) {
+    # Create a unique file ID for each file
+    $fileId = "unique()"
```
@akshatbajetha file id's should not be dynamic as if they are then the contributor would have to manually feed them in the frontend as each time the script runs the id's respective to default profile avatars will change leading the UI to crash if not updated with the new Id's

akshatbajetha replied:
@Aarush-Acharya so how about I just manually add different ID's to all the images being uploaded because right now in the current code, there is same ID of all the images being uploaded which leads to only one image being uploaded on the server.

![{DCBAC4BD-E8CD-4640-8CB5-A10887BA3256}](https://github.com/user-attachments/assets/6630e515-6ff6-4e75-8470-130c3044b6c2)

Aarush-Acharya replied:
@akshatbajetha  Yes this looks great 

Make sure to address the default profile avatar background concern that I rose up in the comment on the main thread

Then push a new commit and will have it merged

Aarush-Acharya commented on init.sh:
```diff
@@ -77,9 +77,17 @@ appwrite project create-variable --key APPWRITE_ENDPOINT --value "http://host.do
+
+for file in "${files[@]}"; do
+    # Create a unique file ID for each file
+    fileId="unique()"
```
make fileId non non dynamic for bash script as well

Aarush-Acharya reviewed (changes requested):
hey @akshatbajetha, good work but please implement the requested changes and also make sure that 
- cream
- amber 
Default avatar images have their respective colours shade in the background like the time, forest and others have.

so that the default profile image for each theme is in the fashion 
avatar colour: primary colour
background colour: primary colour shade

akshatbajetha commented:
Hey @Aarush-Acharya, I did all the requested changes.
- Updated the amber, cream and forest profile image so that the background has primary colour's shade like the other profile images.
- Added the ID's of all the files manually in both ps1 and bash files

Aarush-Acharya commented:
Hey @akshatbajetha, good work

apologies for my late response. I will be merging it now
